File: 389.py
class Solution:
    def findTheDifference(self, s: str, t: str) -> str:
        f=[0] * 26
        for i in s:
            f[ord(i)-ord('a')] += 1
        for i in t:
            f[ord(i)-ord('a')] -= 1
        for i in range(26):
            if (f[i]):
                return chr(i+ord('a'))
        return

File: 389.cpp
class Solution {
public:
    char findTheDifference(string s, string t) {
        int f[26]={0};
        for (int i=0;i<s.size();i++)
            f[s[i]-'a']++;
        for (int i=0;i<t.size();i++)
            f[t[i]-'a']--;
        for (int i=0;i<26;i++)
            if (f[i])
                return (char)(i+'a');
        return ' ';
    }
};
